circle.yml

```diff
@@ -6,6 +6,8 @@ machine:
   # Install ceph to test rados driver & create pool
     - sudo -i ~/distribution/contrib/ceph/ci-setup.sh
     - ceph osd pool create docker-distribution 1
+  # Enable the mystical non-privileged ping socket
+    - sudo sysctl -w net.ipv4.ping_group_range="0 1000"
```

Comment:
Will users have to do this to use this health checker or is this just for unit tests?